modules/dr-recover-lost-control-plane-hosts.adoc

```diff
@@ -29,12 +29,16 @@ This procedure assumes that you have copied a snapshot file called `snapshot.db`
 [core@ip-10-0-143-125 ~]$ export INITIAL_CLUSTER="etcd-member-ip-10-0-143-125.ec2.internal=https://etcd-0.clustername.devcluster.openshift.com:2380"
 ----
 
+.. If the cluster-wide proxy is enabled, be sure that you have exported the `NO_PROXY`, `HTTP_PROXY`, and `HTTPS_PROXY` environment variables.
```

Comment:
@danehans Is this wording okay as is? Is it correct to assume that these variables are already set if the cluster-wide proxy is enabled?

Reply:
@bergerhoffer when cluster-wide egress proxy is configured, those env vars should reside within any platform component's pod(s) requiring egress proxy. I'm unfamiliar with the DR script. I would expect the script to capture the proxy object state before making any changes and ensure that the proxy state matches after the DR procedure is considered successfully complete.

Reply:
@retroflexer I know we're on a host, but not necessarily in a pod, right? wdyt here?

Reply:
I think this is correct. We were running the scripts earlier without `-E` option, and therefore the scripts were not getting the environment variables. Otherwise it is expected that these env variables are already properly set.